playbooks/ansible/roles/samba.setup/tasks/main.yml

```diff
@@ -1,7 +1,25 @@
 ---
+- name: Install samba packages
+  yum:
+    name: samba
+    state: present
+
+- name: Create Samba share directories
+  file:
+    path: "{{ config.paths.mount }}/{{ item.cluster_volume }}"
+    state: directory
+    mode: 0777
+  loop: "{{ samba_shares }}"
```

Comment:
This means that we create a directory under _/mnt_ even if its unused for `glusterfs` and `cephfs.vfs`, right?

Reply:
Yes. I'm even thinking to always mount them even if we later use a VFS module. It's always useful to have access to the data and it shouldn't cause any issue (if it does, it's a bug).

Reply:
Ok.